content/en/account_management/scim/_index.md

```diff
@@ -12,7 +12,7 @@ algolia:
 ---
 
 <div class="alert alert-info">
-SCIM is only available for the Enterprise plan.
+SCIM is available in the Infra Pro plan and all higher-tier plans, including Enterprise.
```

Comment:
How many "higher-tier" plans are there? Would it make sense to list them all out rather than have the reader wonder if they have a higher-tier plan?

Reply:
Hi! That is a great point!

I reached out to the product owners of this doc and I was told that there is indeed I higher tier other than Infra pro, called pro+ but it should not really be mentioned on public documentation.

So for now we are going only with Infrastructure Pro and Infrastructure Entreprise plans.

Reply:
It is now ready to be merged :)